### PR TITLE
[stable/openvpn] fixing typo in  OVPN_K8S_SVC_SUBNET property name

### PR DESCRIPTION
**Pull request by irozet**
**What this PR does / why we need it**:

typo in property name, should be fixed

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #7470 

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/openvpn/templates/config-openvpn.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ data:
 {{ if (.Values.openvpn.OVPN_K8S_POD_NETWORK) (.Values.openvpn.OVPN_K8S_POD_SUBNET) }}
       push "route {{ .Values.openvpn.OVPN_K8S_POD_NETWORK }} {{ .Values.openvpn.OVPN_K8S_POD_SUBNET }}"
 {{ end }}
-{{ if (.Values.openvpn.OVPN_K8S_SVC_NETWORK) (.Values.openvpn.OVPN_SVC_POD_SUBNET) }}
+{{ if (.Values.openvpn.OVPN_K8S_SVC_NETWORK) (.Values.openvpn.OVPN_K8S_SVC_SUBNET) }}
```
Comment (cpanato) [inline stable/openvpn/templates/config-openvpn.yaml:108]: I don't see either the `OVPN_K8S_SVC_NETWORK` and `OVPN_K8S_SVC_SUBNET` in the `values.yaml` file only in the `README.md`
can you please add those in the `values.yaml`?

Comment (irozet) [reply stable/openvpn/templates/config-openvpn.yaml:108]: yes, you are right. Fixed values.yaml

**Pull request merged**